conf/dynaconf_hooks.py

```diff
@@ -33,7 +38,22 @@ def write_cache(path, data):
 
 def read_cache(path):
     logger.info(f'Using settings cache file: {path}')
-    return json.loads(path.read_text())
+    return Box(json.loads(path.read_text()))
+
+
+def config_migrations(settings, data):
+    logger.info('Running config migration hooks')
+    sys.path.append(str(Path(__file__).parent))
+    from conf import migrations
+
+    migration_functions = [
+        mf for mf in getmembers(migrations, isfunction) if mf[0].startswith('migration_')
+    ]
+    for name, func in migration_functions:
+        logger.debug(f'Running {name}')
+        func(settings, data)
+        logger.debug(f'Finished running {name}')
+    logger.info('Finished running config migration hooks')
```

Comment:
can you add some comments here explaining what this is for and where to define the migrations?